### ISSUE TITLE
Error while installing from flatpak

### ISSUE THREAD

**Issue by sg2400** [moneymanagerex/moneymanagerex#6746]
MMEX version:
 - [ ] 1.7.0
 - [X] 1.8.0
 - [ ] 1.8.1
 - [ ] 1.8.2
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [] Windows
 - [ ] Mac OSX
 - [x] Linux 

# Description of the bug

sudo flatpak install flathub org.moneymanagerex.MMEX

gives an error does not install v1.8.0 of MMEX.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

sudo flatpak install flathub org.moneymanagerex.MMEX

Expected result:

Installation of v1.8.0

Actual result:

Gives an error and does not install the program.

# Additional information

If you have a backtrace for a crash or a warning, paste it here.

sudo flatpak install flathub org.moneymanagerex.MMEX
[sudo] password for root: 
Looking for matches…

org.moneymanagerex.MMEX permissions:
    ipc                  network      fallback-x11      wayland      x11     dri
    file access [1]

    [1] /media, /mnt, /run/media, home

        ID                               Branch          Op          Remote           Download
 1. [✗] org.moneymanagerex.MMEX          stable          i           flathub          7.3 MB / 21.0 MB

Error: While pulling app/org.moneymanagerex.MMEX/x86_64/stable from remote flathub: opcode set-read-source: Opening content object 4b05fbc48808f58d3e7dea4f34510dddaa15ea6ab30b772e05bd4ec717fa9d07: Couldn't find file object '4b05fbc48808f58d3e7dea4f34510dddaa15ea6ab30b772e05bd4ec717fa9d07'
error: Failed to install org.moneymanagerex.MMEX: While pulling app/org.moneymanagerex.MMEX/x86_64/stable from remote flathub: opcode set-read-source: Opening content object 4b05fbc48808f58d3e7dea4f34510dddaa15ea6ab30b772e05bd4ec717fa9d07: Couldn't find file object '4b05fbc48808f58d3e7dea4f34510dddaa15ea6ab30b772e05bd4ec717fa9d07

**Comment by whalley** [comment]
@sg2400 Is this still an issue with v1.9.0?

**Comment by sg2400** [comment]
I don't know as I gave up on using it on Linux since that bug. I think it should be closed as it is no longer relevant.
